nav2_route/include/nav2_route/graph_loader.hpp

```diff
@@ -69,9 +70,18 @@ class GraphLoader
     std::string filepath = "");
 
 protected:
+
+  /**
+   * @brief Transform the graph coordinates to the route frame
+   * @param[in/out] graph The graph to be transformed
+   * @return True if transform was successful
+   */
+  bool transformGraph(Graph &graph);
+
   std::string route_frame_, graph_filepath_;
   std::shared_ptr<tf2_ros::Buffer> tf_;
   rclcpp::Logger logger_{rclcpp::get_logger("GraphLoader")};
+  nav2_util::LifecycleNode::SharedPtr node_;
```

Comment:
Don't store this - it doesn't look like you actually use it outside of the constructor anyway. If you must, store it as a weakptr, but I don't think you do